### DISCUSSION TITLE
Continuous download or way to set dates

### DISCUSSION THREAD

**Discussion by rayzinbran** [davidteather/TikTok-Api#763] [Q&A]
Is there a way to set this API to download continuously rather than by count? I'm helping out with a research project and we would need to build an ever-changing database as the videos update over time and would rather have a way to set it and forget it somehow. Either b running it continuously or by somehow saving our spot / having it check what was downloaded and pickup from the next item.

**Comment by davidteather** [comment]
Here's one approach you could take
1. Log video ids in a database
2. Every x minutes have a background job to call TikTokApi and update in your database

There's not a way to directly do this
